test/display_test.py

```diff
@@ -200,9 +206,65 @@ def todo_test_gl_get_attribute(self):
         # attributes. See pygame.display.gl_set_attribute() for a list of
         # valid flags.
         #
-
-        self.fail()
-
+ 	
+        # assign SDL1-supported values with gl_set_attribute
+        pygame.display.gl_set_attribute(pygame.GL_ALPHA_SIZE,8)
+        pygame.display.gl_set_attribute(pygame.GL_DEPTH_SIZE,32)
+        pygame.display.gl_set_attribute(pygame.GL_STENCIL_SIZE,0)
+        pygame.display.gl_set_attribute(pygame.GL_ACCUM_RED_SIZE,16)
+        pygame.display.gl_set_attribute(pygame.GL_ACCUM_GREEN_SIZE,16)
+        pygame.display.gl_set_attribute(pygame.GL_ACCUM_BLUE_SIZE,16)
+        pygame.display.gl_set_attribute(pygame.GL_ACCUM_ALPHA_SIZE,16)
+        pygame.display.gl_set_attribute(pygame.GL_MULTISAMPLEBUFFERS,0)
+        pygame.display.gl_set_attribute(pygame.GL_MULTISAMPLESAMPLES,0)
+        pygame.display.gl_set_attribute(pygame.GL_STEREO,0)
+
+        # assign SDL2-supported values with gl_set_attribute (if applicable)
+        if(SDL2):
+            pygame.display.gl_set_attribute(pygame.GL_ACCELERATED_VISUAL,0)
+            pygame.display.gl_set_attribute(pygame.GL_CONTEXT_MAJOR_VERSION,1)
+            pygame.display.gl_set_attribute(pygame.GL_CONTEXT_MINOR_VERSION,1)
+            pygame.display.gl_set_attribute(pygame.GL_CONTEXT_FLAGS, 0)
+            pygame.display.gl_set_attrubute(pygame.GL_CONTEXT_PROFILE_MASK,0)
+            pygame.display.gl_set_attribute(pygame.GL_SHARE_WITH_CURRENT_CONTEXT,0)
+            pygame.display.gl_set_attribute(pygame.GL_CONTEXT_RELEASE_BEHAVIOR,1)
+            pygame.display.gl_set_attribute(pygame.GL_FRAMEBUFFER_SRGB_CAPABLE,0)
+
+        # call set_mode with OPENGL flag
+        screen = display.set_mode((0,0), pygame.OPENGL)
+
+        # test SDL1 flags using gl_get_attribute
+        self.assertEqual(pygame.display.gl_get_attribute(pygame.GL_ALPHA_SIZE),8)
+        self.assertEqual(pygame.display.gl_get_attribute(pygame.GL_DEPTH_SIZE),32)
+        self.assertEqual(pygame.display.gl_get_attribute(pygame.GL_STENCIL_SIZE),0)
```

Comment:
Test fails when run locally on windows here:

```
======================================================================
FAIL: test_gl_get_attribute (__main__.DisplayModuleTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:/Users/Dan/Programming/pygame/test/display_test.py", line 239, in test_gl_get_attribute
    self.assertEqual(pygame.display.gl_get_attribute(pygame.GL_STENCIL_SIZE), 0)
AssertionError: 8 != 0

----------------------------------------------------------------------
```

Perhaps this attribute can't be set to 0 ? Try 8.